manifest-c10s.yaml

```diff
@@ -0,0 +1,30 @@
+# Manifest for CentOS Stream CoreOS 10
+
+rojig:
+  license: MIT
+  name: scos
+  summary: CentOS Stream CoreOS 10
+
+variables:
+  osversion: "c10s"
+  inherit_tier_x: true
+
+# Include manifests common to all RHEL and CentOS Stream versions
+include:
+  - common.yaml
+  - common-el10.yaml
+
+repos:
+  - c10s-baseos
+  - c10s-appstream
+
+automatic-version-prefix: "10.<date:%Y%m%d%H%M>"
```

Comment:
```suggestion
automatic-version-prefix: "10.0.<date:%Y%m%d>"
```

Reply:
Compare this with what we are doing for c9s. @jlebon chose 9.0.<date> intentionally.

Reply:
Hum, indeed., will do